samples/net/coaps_server/prj_qemu_x86.conf

```diff
@@ -14,7 +14,7 @@ CONFIG_NET_PKT_TX_COUNT=14
 CONFIG_NET_BUF_RX_COUNT=30
 CONFIG_NET_BUF_TX_COUNT=30
 CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=3
-CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=2
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
IMHO we should always have working config so overestimating the requirements would be better. The application developer needs to tweak values anyways if he/she wants to optimize the resource usage. Of course finding the good values for various config options can be tricky.